tell/visualization.py

```diff
@@ -125,13 +125,6 @@ def plot_state_load_time_series(state, data_input_dir, year_to_plot, save_images
         plt.savefig((image_output_dir + year_to_plot + '/' + 'TELL_State_Hourly_Loads_' + state_name + '_' + year_to_plot + '.png'),
                     dpi=image_resolution, bbox_inches='tight')
 
-        def plot_state_load_duration_curve(state, state_hourly_load_df, year_to_plot, save_images, image_resolution,
```

Comment:
this didn't appear to be used anywhere and was mis-indented, so i just axed it -- you may actually need or want it somewhere 👼